services/one-stream/package/buildAnyPackage.js

```diff
@@ -15,7 +15,6 @@ const SEATOOL_TO_ONEMAC_STATUS = {
   [Workflow.SEATOOL_STATUS.TERMINATED]: Workflow.ONEMAC_STATUS.TERMINATED,
   [Workflow.SEATOOL_STATUS.PENDING_CONCURRENCE]:
     Workflow.ONEMAC_STATUS.PENDING_CONCURRENCE,
-  [Workflow.SEATOOL_STATUS.UNSUBMITTED]: Workflow.ONEMAC_STATUS.UNSUBMITTED,
```

Comment:
Yeah.... this is intentional... apparently SEA Tool has an Unsubmitted status, but it is no longer used and the spreadsheet of actions specifically called it out as unsupported... I had left in the conversion because I thought we might need it, but then, because Unsubmitted was not in the "approved status list" for the filters, the package was getting removed from the dashboard if it entered Unsubmitted... I figured the best solution was to remove it as an "Accepted status" so that the package status would not update and the package would still show on the dashboard with an older status.